api/src/main/java/marquez/db/OpenLineageDao.java

```diff
@@ -117,6 +119,34 @@ void createLineageEvent(
   LIMIT :limit""")
   List<LineageEvent> getAllLineageEventsAsc(ZonedDateTime before, ZonedDateTime after, int limit);
 
+  @SqlQuery(
+      """
+    SELECT
+      run_uuid,
+      JSON_AGG(event -> 'run' -> 'facets' ORDER BY event_time) AS facets
```

Comment:
if we were able to merge #2355, it would be great to make use of `run_facets_view`.

Reply:
@JDarDagran we were able to ;)